spec/helpers/csv_name_helper_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe CSVNameHelper, type: :helper do
+  around do |example|
+    Timecop.freeze(2020, 6, 1, 12) do
+      example.run
+    end
+  end
+
+  let(:export_name) { 'extremely-cool-data' }
+  let(:cycle_years) { [0, 2000] }
```

Comment:
Shouldnt this be [2000] only?

Reply:
For some reason the application data export adds a 0 to the cycle years array - this was a quick fix, but I'll do a bit of debugging to see why it's doing this 
![image](https://user-images.githubusercontent.com/25597009/140507292-fe709222-ffdb-4405-ba72-6e937a877185.png)